Sources/Redis/CRUD.swift

```diff
@@ -1,6 +1,20 @@
 import Async
 
 extension RedisClient {
+    /// Authorizes the client
+    ///
+    /// - returns: A future that will be completed (or failed) when the client is authorized
+    @discardableResult
+    public func authorize(with password: String) -> Future<RedisClient> {
```

Comment:
Would returning `Future<Void>` make more sense here?

Reply:
Yeah I think it would make more sense.  I forget the reason why I made it return a RedisClient.